### ISSUE TITLE
Inputs with non-interactive icon - needs aria-hidden

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1232]
CENG_3303

Docs: https://react-magma.cengage.com/version/4.3.0/api/input/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

When an icon is placed within an input and is not an interactive element, it would be considered decorative and should be hidden from assistive technology.

Add aria-hidden="true" to icons within text inputs when the icon is not an interactive element.
